flixel/system/debug/Console.hx

```diff
@@ -112,7 +112,7 @@ class Console extends Window
 		_input.addEventListener(FocusEvent.FOCUS_OUT, onFocusLost);
 		_input.addEventListener(KeyboardEvent.KEY_DOWN, onKeyPress);
 		
-		#if (cpp || neko) // workaround for broken TextField focus on native
```

Comment:
And yes, this does not fix input text field support in any case. It just gets the key events back to where they should be. (Though as a bonus, the focus in/out from clicking on and off the debugger console seems to have the right behavior now with this fix, even if you can't type stuff in yet)